data/dineshlalam15.json

```
@@ -12,6 +12,16 @@
       "name": "Follow me on GitHub",
       "url": "https://github.com/dineshlalam15",
       "icon": "FaGithub"
+    },
+    {
+      "name": "Hey! Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/dinesh-kumar-lalam-2584a8207/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Look into my blogs here :)",
+      "url": "https://hashnode.com/@dineshlalam15",
+      "icon": "FaHashnode"
```

You probably be using an invalid icon here, it should be `SiHashnode`.

```suggestion
      "icon": "SiHashnode"
```